src/test.py

```diff
@@ -74,11 +76,13 @@ def work():
 def task_queue_integration_tests():
     def work():
         print("Running task queue integration tests")
+        yt = YTClient('https://mrc-ide.myjetbrains.com/youtrack/',
+                      token=os.environ["YOUTRACK_TOKEN"])
```

Comment:
this has been added to the build agents here: https://github.com/reside-ic/reside-hyperv-scripts/pull/20